src/Symfony/Component/HttpKernel/Controller/ControllerResolver.php

```diff
@@ -47,23 +43,27 @@ public function getController(Request $request)
         }
 
         if (is_array($controller)) {
+            if (isset($controller[0]) && is_string($controller[0])) {
+                $controller[0] = $this->instantiateController($controller[0]);
```

Comment:
wouldn't this break support for using static methods as controller ?

Reply:
Yeah I also thought about this. We don't have tests using static methods and `class::method` already had the problem of not supporting static methods while that would a valid php callable.
But I already have a solution for this in my mind that will support both cases.

Reply:
implemented